components/preparing_workspace/preparing_workspace.tsx

```diff
@@ -364,22 +362,6 @@ export default function PreparingWorkspace(props: Props) {
             }
         };
 
-        // prepare task list to open in the next view
-        dispatch(savePreferences(user.id, [
-            {
-                user_id: user.id,
-                category: OnboardingTaskCategory,
-                name: OnboardingTaskList.ONBOARDING_TASK_LIST_SHOW,
-                value: 'true',
-            },
-            {
-                user_id: user.id,
-                category: OnboardingTaskCategory,
-                name: OnboardingTaskList.ONBOARDING_TASK_LIST_OPEN,
-                value: 'true',
-            },
-        ]));
-
```

Comment:
Is this no longer needed to open the task list for the first admin?

Reply:
no, all users including first admins will get the initial values (show/open) set.